README.md

```diff
@@ -1,3 +1,77 @@
 # cluster-api-actuator-pkg
 
 Shared packages for Cluster API actuators.
+
+## Runing the e2e tests locally using minikube
```

Comment:
how would this work without running the mao which is a requirement for cao and the clusteroperator CRD?

Reply:
> how would this work without running the mao which is a requirement for cao and the clusteroperator CRD?

Made the comment more specific - now only caters for autoscaler tests.